PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py

```diff
@@ -18,3 +18,25 @@
     patJets
     )
 makePatJets = cms.Sequence(makePatJetsTask)
+
+from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import * #pfImpactParameterTagInfos
+from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import * #pfSecondaryVertexTagInfos
+from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderTagInfos_cfi import * #pfInclusiveSecondaryVertexFinderTagInfos
+from RecoBTag.Combined.deepFlavour_cff import * #pfDeepFlavourTask
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+#make a copy to avoid labels and substitution problems
+_makePatJetsCopy = makePatJets.copy()
+
+run2_miniAOD_80XLegacy.toReplaceWith(
+    makePatJets, 
+    cms.Sequence(
```

Comment:
Hi @slava77 , I tried to test the code on ```/store/mc/RunIISummer16DR80Premix/TT_TuneCUETP8M2T4_13TeV-powheg-pythia8/AODSIM/PUMoriond17_backup_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/00000/004973E8-26BA-E611-93E0-A0000420FE80.root``` 
But I got the following error, unrelated to my changes:

```
----- Begin Fatal Exception 08-Jun-2017 10:46:41 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 190561 event: 30527714 stream: 0
   [1] Running path 'MINIAODSIMoutput_step'
   [2] Prefetching for module PoolOutputModule/'MINIAODSIMoutput'
   [3] Prefetching for module PATElectronSlimmer/'slimmedElectrons'
   [4] Prefetching for module PATElectronSelector/'selectedPatElectrons'
   [5] Prefetching for module PATElectronProducer/'patElectrons'
   [6] Prefetching for module CITKPFIsolationSumProducerForPUPPI/'egmElectronPUPPIIsolation'
   [7] Calling method for module PATPackedCandidateProducer/'packedPFCandidates'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: edm::ValueMap<bool>
Looking for module label: chargedHadronPFTrackIsolation
Looking for productInstanceName:

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```

I am not sure though that it is the right production to test it on. Can you suggest a suitable input file?

I am also a bit puzzled by your comment. I first changed the task and, at least I got it that way, you wanted me to change the sequence. Now you say the very opposite. Tasks are still largely a mystery to me, can you be as detailed as possible with the changes you want? Thanks